src/GafferSceneUI/AttributeInspector.cpp

```diff
@@ -245,9 +248,47 @@ IECore::ConstObjectPtr AttributeInspector::value( const GafferScene::SceneAlgo::
 	return nullptr;
 }
 
-IECore::ConstObjectPtr AttributeInspector::fallbackValue( const GafferScene::SceneAlgo::History *history ) const
+IECore::ConstObjectPtr AttributeInspector::fallbackValue( const GafferScene::SceneAlgo::History *history, std::string &description ) const
 {
-	return history->scene->fullAttributes( history->context->get<ScenePlug::ScenePath>( ScenePlug::scenePathContextName ) )->member( m_attribute );
+	ScenePlug::PathScope pathScope( Context::current() );
+	ScenePlug::ScenePath currentPath( history->context->get<ScenePlug::ScenePath>( ScenePlug::scenePathContextName ) );
+	ConstObjectPtr inheritedAttribute;
+	if( currentPath.size() > 1 )
```

Comment:
Took me a little while to convince myself that although this is the same test as used in the SetMembershipInspector, it is OK here, because we don't allow attributes on the root location.

Reply:
Added a comment to clarify this and squashed to d1c5ac3.